packages/commons-server/src/libs/templating-helpers/data-helpers.ts

```diff
@@ -83,7 +83,11 @@ export const DataHelpers = function (
 
         // ensure a value was found at path
         const foundValue = objectGet(value, path);
-        value = foundValue !== undefined ? foundValue : value;
+        if (parameters.length > 1 && parameters[2]) {
+          value = foundValue !== undefined ? foundValue : '';
+        } else {
+          value = foundValue !== undefined ? foundValue : value;
+        }
```

Comment:
Could you please put this in a separated PR? I prefer keeping each PR focused on one topic. 
Also, this is clearly a discrepancy between how data/dataRaw and body/bodyRaw helpers are working. So, the change is needed but I would rather think this thoroughly in a separated task. (I can also handle it, in which case you can remove it altogether)

Reply:
So do you want I add this behavior on:
- data/dataRaw
- body/bodyRaw
and that I create a specific new PR for this change?

Reply:
Don't worry, I will do it.
But anyway, PR may be merged but the next release will only occur around January.

Reply:
tnx, keep us updated if it is possible